### PR TITLE
config: relocate graph.backend and analyzer.storage

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

**Comment by nplanel** [comment]
run tests

**Comment by nplanel** [comment]
run skydive-devstack-tests

**Comment by nplanel** [comment]
run skydive-packaging-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-python-tests

**Comment by nplanel** [comment]
run skydive-python-tests

**Comment by nplanel** [comment]
run skydive-create-vagrant-boxes

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-devstack-tests

File: config/config.go [hunk 1]
```diff
@@ -89,12 +93,10 @@ func init() {
 	cfg.SetDefault("analyzer.packet_injection_enabled", true)
 	cfg.SetDefault("analyzer.replication.debug", false)
 	cfg.SetDefault("analyzer.ssh_enabled", false)
-	cfg.SetDefault("analyzer.storage.bulk_insert", 100)
-	cfg.SetDefault("analyzer.storage.bulk_insert_deadline", 5)
-	cfg.SetDefault("analyzer.storage.index_age_limit", 0)
-	cfg.SetDefault("analyzer.storage.index_entries_limit", 0)
-	cfg.SetDefault("analyzer.storage.indices_to_keep", 0)
-	cfg.SetDefault("analyzer.storage.max_flow_buffer_size", 100000)
+	cfg.SetDefault("analyzer.flow.bulk_insert", 100)
```
Comment (lebauce) [inline config/config.go]: Can you please keep the alphabetical order ? :-)

**Comment by nplanel** [comment]
run skydive-packaging-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-devstack-tests

**Comment by nplanel** [comment]
run skydive-coverage

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Comment by nplanel** [comment]
run skydive-create-vagrant-boxes

**Pull request merged**